docs/developer/procedures/stable.rst

```diff
@@ -39,8 +47,7 @@ Do you have a Jira issue? Chances are any change worth doing on the branch has a
 Applying your Change to master
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is nice to do this after every fix, or you may want to save up a couple.
-Please try to apply your fixes to trunk before the next trunk Milestone release (we like to release the best code we can - and that includes your fixes).
+If you've applied a fix to the stable branch, and it can be ported to master, then do so. Please try to apply your fixes to master before the next Release Candidate (we like to release the best code we can - and that includes your fixes).
```

Comment:
Hmm... how one gets to the fix is not the business of our documentation, and agree it's common to reproduce and fix on whatever is the target version first, but mind, PRs are only accepted on master first, and then backported on stable.

Reply:
I tried to clean up the existing docs a bit here (they already went into process a bit, but felt outdated). It might make more sense to restructure the order of them so that the "Apply to master" section comes before "Apply to stable" to drive the point home a bit more, but that starts conflicting with the ostensible purpose of the page - "Working on the stable branch".
Do you think changes are needed here?

Also, there are rare cases in which we do accept PRs on a stable branch before master, when a fix cannot be applied to master (either because it was fixed on master by a non-backportable dependency change, or affects a module that no longer exists), but that is nitpicking a bit.

Reply:
The cases are there indeed, but very, very rare.

About the contents of the page, I believe we just need to clarify that committig on the master branch is a pre-requisite to merging on the stable branch, unless there is a reason making the commit to master un-needed, or impossible.

Reply:
I've striped out some of the process suggestions and made it more clear that you must make a PR against master before stable.
I also tried to make the existing instructions a bit more consistent, and removed several lines that suggested pushing directly to master (replacing those with "open a pull request" instead). Hopefully everything is still pretty clear.

The recommendations around API changes remain the same.